### PR TITLE
Character and background switch

### PR DESCRIPTION
**Pull request by strawberriesandcheese**

### CODE REVIEW

File: VNScripts/CharacterView.gd [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+var characterMap = {
+	"thief": preload("res://RealElvenThief.tscn"),
+	"cat": preload("res://CatDude.tscn")
+}
+
+var exampleCharacterList1 = ["thief", "cat"]
+var exampleCharacterList2 = ["cat", "thief"]
+
+onready var positions = [
+
```
Comment (Junber) [inline VNScripts/CharacterView.gd]:
```suggestion
```

File: VNScripts/CharacterView.gd [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+var exampleCharacterList1 = ["thief", "cat"]
+var exampleCharacterList2 = ["cat", "thief"]
+
+onready var positions = [
+	
+	[
+		#positions[0][0]
+		$"1CharacterPosition1".position
```
Comment (Junber) [inline VNScripts/CharacterView.gd:16]: Maybe call this 1CharacterPosition0 for less confusion? Then you can also drop to comments here, I think

File: VNScripts/CharacterView.gd [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+var currentCharacters = []
+
+var characterMap = {
+	"thief": preload("res://RealElvenThief.tscn"),
+	"cat": preload("res://CatDude.tscn")
+}
+
+var exampleCharacterList1 = ["thief", "cat"]
+var exampleCharacterList2 = ["cat", "thief"]
```
Comment (Junber) [inline VNScripts/CharacterView.gd:11]: Remove

File: VNScripts/CharacterView.gd [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+onready var backgroundMap = {
+	"thunderstorm": $Background1,
+	"field": $Background2,
+	"mountain": $Background1
+}
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	change_background("field")
```
Comment (Junber) [inline VNScripts/CharacterView.gd:34]: Maybe export the first background

File: VNScripts/CharacterView.gd [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+func add_character(character: String, position):
+	var new_character = characterMap[character].instance()
+	new_character.position = position
+	add_child(new_character)
+	currentCharacters.append(new_character)
+	return new_character
+
+func remove_characters():
+	for character in currentCharacters:
+		remove_child(character)
```
Comment (Junber) [inline VNScripts/CharacterView.gd]:
```suggestion
		character.queue_free()
```
Else the character won't be deleted

File: VNScripts/CharacterView.gd (hunk 4 above)
Comment (strawberriesandcheese) [reply VNScripts/CharacterView.gd:34]: neat idea

**Pull request merged**